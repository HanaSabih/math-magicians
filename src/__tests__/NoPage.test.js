import renderer from 'react-test-renderer';
import NoPage from '../components/NoPage';

it('renders correctly', () => {
  const testQuote = renderer.create(<NoPage />).toJSON();
  expect(testQuote).toMatchSnapshot();
});