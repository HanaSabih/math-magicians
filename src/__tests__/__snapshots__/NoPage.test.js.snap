// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    404
  </h1>
</div>
`;
